csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -476,9 +484,9 @@ public void AddDefault(string facet, IObject servant)
         }
 
         /// <summary>Creates a proxy for the object with the given identity and facet. If this object adapter is
-        /// configured with an adapter ID, creates an indirect proxy that refers to the adapter ID. If a replica group
-        /// ID is also defined, creates an indirect proxy that refers to the replica group ID. Otherwise, if no adapter
```

Comment:
accepts _only_ non-secure connections?

Since with ice1, !IsAlwaysSecure means is always non-secure.

Reply:
Not sure what line your referencing here